### DISCUSSION TITLE
anvas in HTML

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#44] [Q&A] [answered]
What is the use of Canvas in HTML?

**Comment by Bunny77K9** [comment, marked as answer]
Canvas was added onto HTML5 to give users the ability to draw graphics on the go, using JavaScript. There are a variety of methods in <canvas> to allow for the drawing of paths, circles, boxes, images, and more.

Next up on this top Web Developer interview questions and answers, let us understand the difference between canvas and SVG.
